### PR TITLE
Make RTL support better.

### LINKED ISSUE #961: Inconsistent interface with Arabic
### Checklist / 检查清单

- [X] No one has submitted a similar or identical bug report before. / 之前没有人提交过类似或相同的 bug report。
- [X] I'm using the latest version of LibChecker. / 我正在使用 LibChecker 的最新版本。

### LibChecker version / LibChecker 版本

Latest CI / 最新 CI 版

### Bug description / Bug 描述

In the sorting list as in the picture below some names do not appear and some are displayed incorrectly, the problem is in the display in Arabic. 

IMG_20231019_095254_350

### Expected behavior / 预期行为

None

### Actual behavior / 实际行为

None

### Steps to reproduce / 复现步骤

None

### UI / OS

On UI

### Android Version / Android 版本

11

### Additional info / 额外信息

Even the application is not compatible with Arabic because the lists in Arabic must be from right to left. In these two pictures illustration, number one for the application, number two example

1. 
IMG_20231019_100928_611

2.  Example
IMG_20231019_100925_275

### PR DESCRIPTION
**Pull request by zhaobozhen**
Closes #961

### CODE REVIEW

File: app/src/main/kotlin/com/absinthe/libchecker/features/applist/detail/ui/view/AppPropItemView.kt [hunk 1]
```diff
@@ -20,8 +22,10 @@ class AppPropItemView(context: Context) : AViewGroup(context) {
       R.style.TextView_SansSerif
     )
   ).apply {
-    layoutParams =
-      LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, -1)
+    layoutParams = LayoutParams(
+      ViewGroup.LayoutParams.WRAP_CONTENT,
+      -1
```
Comment (qhy040404) [inline app/src/main/kotlin/com/absinthe/libchecker/features/applist/detail/ui/view/AppPropItemView.kt]:
```suggestion
      ViewGroup.LayoutParams.MATCH_PARENT
```

File: app/src/main/kotlin/com/absinthe/libchecker/features/snapshot/ui/adapter/SnapshotAdapter.kt [hunk 2]
```diff
@@ -131,7 +131,7 @@ class SnapshotAdapter(private val cardMode: CardMode = CardMode.NORMAL) : Highli
           item.packageSizeDiff.new
         )
         val diffText = buildSpannedString {
-          append(LCAppUtils.getDiffString(sizeDiff, bytesDiff, isNewOrDeleted, highlightDiffColor = highlightDiffColor))
+          append(LCAppUtils.getDiffString(sizeDiff, bytesDiff, isNewOrDeleted, highlightDiffColor = highlightDiffColor, diff2Suffix = " Bytes"))
```
Comment (qhy040404) [inline app/src/main/kotlin/com/absinthe/libchecker/features/snapshot/ui/adapter/SnapshotAdapter.kt]: why here

File: app/src/main/kotlin/com/absinthe/libchecker/features/snapshot/ui/adapter/SnapshotAdapter.kt
Comment (qhy040404) [inline app/src/main/kotlin/com/absinthe/libchecker/features/snapshot/ui/adapter/SnapshotAdapter.kt:1]: Screenshot_2023-11-29-21-57-30-787_com absinthe libchecker debug 1
App name diff, API diff, `ARROW`.
Randomly versionInfo diff.

**Review by qhy040404 — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/com/absinthe/libchecker/features/snapshot/ui/adapter/SnapshotAdapter.kt (hunk 2 above)
Comment (zhaobozhen) [reply app/src/main/kotlin/com/absinthe/libchecker/features/snapshot/ui/adapter/SnapshotAdapter.kt]: I only want to highlight diff of numbers but not the word `Bytes`

File: app/src/main/kotlin/com/absinthe/libchecker/features/snapshot/ui/adapter/SnapshotAdapter.kt [hunk 3]
```diff
@@ -33,7 +33,7 @@ import java.text.SimpleDateFormat
 import java.util.Locale
 
 const val ARROW = "→"
-
+const val ARROW_REVERT = "←"
```
Comment (qhy040404) [inline app/src/main/kotlin/com/absinthe/libchecker/features/snapshot/ui/adapter/SnapshotAdapter.kt:36]:
```kotlin
val ARROW: String get() = if (GlobalValues.locale.layoutDirection == View.LAYOUT_DIRECTION_RTL) "←" else "→"
```

File: app/src/main/kotlin/com/absinthe/libchecker/utils/IcuUtils.kt
Comment (qhy040404) [inline app/src/main/kotlin/com/absinthe/libchecker/utils/IcuUtils.kt:1]: May not be effective
image

Comment (qhy040404) [reply app/src/main/kotlin/com/absinthe/libchecker/utils/IcuUtils.kt:1]:
```kotlin
androidx.core.text.BidiFormatter.getInstance().isRtl(text)
```

**Review by qhy040404 — APPROVED** [review]
开摆

**Pull request merged**